lizmap/modules/lizmap/classes/lizmapServices.class.php

```diff
@@ -345,6 +349,32 @@ public function getRootRepositories()
         return $rootRepositories;
     }
 
+    /**
+     * Get the path where the inspection data generated by qgis-project-validator
+     * tool are stored. If not found, use the rootRepositories folder.
+     *
+     * @return string Path of the folder containing the inspection data
+     */
+    public function getQgisProjectsInspectionDataFolder()
+    {
+        $qgisProjectsInspectionDataFolder = $this->qgisProjectsInspectionDataFolder;
+
+        if ($qgisProjectsInspectionDataFolder != '') {
+            // if path is relative, get full path
+            if ($qgisProjectsInspectionDataFolder[0] != '/' and $qgisProjectsInspectionDataFolder[1] != ':') {
+                $qgisProjectsInspectionDataFolder = realpath($this->varPath.$qgisProjectsInspectionDataFolder);
+            }
+            // add a trailing slash if needed
+            if (!preg_match('#/$#', $qgisProjectsInspectionDataFolder) && $qgisProjectsInspectionDataFolder !== false) {
```

Comment:
It is faster I guess: `$qgisProjectsInspectionDataFolder = rtrim($qgisProjectsInspectionDataFolder, '/').'/';`